cookbooks/cdo-varnish/libraries/helpers.rb

```diff
@@ -110,11 +110,9 @@ def process_vary(behavior, _)
   behavior[:headers].each do |header|
     out << set_vary(header, 'beresp')
   end
-  case behavior[:cookies]
-  when 'all'
+  if behavior[:cookies] == 'all'
     out << set_vary('Cookie', 'beresp')
-  when 'none'
-  else
+  elsif behavior[:cookies] != 'none'
```

Comment:
I think if we don't want empty `when` clauses then `if/else` is the best way. It's pretty readable, though unfortunately `behavior[:cookies]` is repeated now :(